README.md

```diff
@@ -2,3 +2,53 @@
 
 Data Base abstractions to be used in applications.
 These abstractions are not only meant to be used in applications built on [Tendermint](https://github.com/tendermint/tendermint), but can be used in a variety of applications.
+
+### Minimum Go Version
+
+Go 1.13+
+
+## Supported Databases
+
+- [GolevelDB](https://github.com/syndtr/goleveldb)
+- [ClevelDB](https://github.com/google/leveldb)
+- [BoltDB](https://github.com/etcd-io/bbolt)
+- [GoRocksDB](https://github.com/tecbot/gorocksdb)
+- [MemDB](./mem_db.go)
+- [RemoteDB](#remotedb)
+
+
+## Using Databases
+
+### GolevelDB
+
+To use goleveldb there is no need to install anything and you can run the tests by simply running `make test`
+
+### ClevelDB
+
+To use cleveldb leveldb must be installed on your machine. Please use you local package manager (brew, snap) to install the db. Once you have installed it you can run tests with `make test-cleveldb`.
+
+### BoltDB
+
+To use BoltDB, you must have it installed on your machine. You can install it by running:
```

Comment:
Hmm, is this true? Yes, you can install the binary to manually do compaction, view something, but I thought it's optional...

Reply:
just tested it, you are right.

Reply:
go runtime won't include code in the end binary (`tendermint`)